.github/workflows/pr-checks.yaml

```diff
@@ -34,6 +34,28 @@ jobs:
           args: --timeout 10m0s
           go-version: ${{ env.go-version }}
 
+  jsonnet-fmt:
+    runs-on: ubuntu-latest
+    name: jsonnet-fmt
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use tools cache
+        uses: ./.github/tools-cache
+
+      - run: make jsonnet-fmt && git diff --exit-code
+
+  jsonnet-lint:
+    runs-on: ubuntu-latest
+    name: jsonnet-lint
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use tools cache
```

Comment:
we should include the `go-cache` before tools cache since go mods will be reused.